libraries/AP_SmartRTL/AP_SmartRTL.cpp

```diff
@@ -166,6 +166,19 @@ bool AP_SmartRTL::pop_point(Vector3f& point)
     return true;
 }
 
+//Sets home and resets path when position is ok, ahrs has origin, and already armed. Avoids bad position GCS message
+void AP_SmartRTL::save_home(bool position_ok)
```

Comment:
The indent of this method is 2. 4 White space is a code style.

Reply:
Ah sorry about that. Will fix